dataactvalidator/config/sqlrules/fabs37_detached_award_financial_assistance_2.sql

```diff
@@ -17,16 +17,16 @@ SELECT
     action_type,
     correction_late_delete_ind,
     action_date
-FROM detached_award_financial_assistance_d37_2_{0} AS dafa
+FROM detached_award_financial_assistance_fabs37_2_{0} AS dafa
 WHERE dafa.action_type IN ('B', 'C', 'D')
     AND ((dafa.correction_late_delete_ind != 'C')
         or (dafa.correction_late_delete_ind is null)
     )
     AND dafa.row_number IN (
         SELECT DISTINCT sub_dafa.row_number
-        FROM detached_award_financial_assistance_d37_2_{0} AS sub_dafa
+        FROM detached_award_financial_assistance_fabs37_2_{0} AS sub_dafa
             JOIN cfda_program AS cfda
-            ON (sub_dafa.cfda_number IS NOT DISTINCT FROM to_char(cfda.program_number, 'FM99D999')
+            ON (sub_dafa.cfda_number = to_char(cfda.program_number, 'FM00.000')
```

Comment:
Is there a reason that "FM99D999" was replaced with "FM00.000"? I can't find it in the acceptance criteria

Reply:
I had changed it in part 1 because of a bug where an entry of `12.340` would be read as `12.34` and that's not correct. The 9 indicates a digit and the 0 indicates a 0-filled value so it's ALWAYS that length. I realized while I was fixing this one for speed that I hadn't fixed the formatting. I don't remember why I changed D to . but they both mean a decimal point (I think the . is used more for 0-filled? There was research involved a month or two ago) so I figured we should be consistent.